crates/ln-dlc-node/src/node.rs

```diff
@@ -798,10 +773,17 @@ pub(crate) fn app_config() -> UserConfig {
         },
         channel_handshake_limits: ChannelHandshakeLimits {
             max_minimum_depth: 1,
+            trust_own_funding_0conf: true,
```

Comment:
🙃 Maybe a comment for this option would also be useful

Reply:
well, this is actually just the default config and does not change anything. 🙈